src/components/ButtonComponents/OperatorButtons/OperatorButton.js

```diff
@@ -1,9 +1,11 @@
 import React from "react";
 
-const OperatorButton = () => {
+const OperatorButton = (props) => {
   return (
-    <>
-      {/* Display a button element rendering the data being passed down from the parent container on props */}
-    </>
+    <button className = "operatorbutton all">
+      {props.operator/* Display a button element rendering the data being passed down from the parent container on props */}
```

Comment:
We are going to get an issue here. That is because the operators array contains objects, not strings. React can't directly insert an object into our document. So, this would need to be `props.operator.char`.